test/context.test.ts

```diff
@@ -38,11 +38,11 @@ export function dummyCtx({ userInput, language, noChat }: {
 }) {
     const u = { id: 42, first_name: "yo", language_code: language } as User;
     const c = { id: 100, type: "private" } as Chat;
-    const m = {
+    const m = noChat ? undefined : ({
```

Comment:
why are we setting `message` to `undefined` if `noChat` is specified?

Reply:
`dummyCtx` is a helper function for testing purposes. The `noChat` flag was introduce to test lines related to updates with no message / chat, like [this one](https://app.codecov.io/gh/grammyjs/commands/pull/31/blob/src/context.ts#L69)

Reply:
I see, but this is making the message undefined. I think we should at least change it to `noMessage`, just to avoid futur confusion